formatters/formatters.go

```diff
@@ -0,0 +1,8 @@
+package formatters
+
+import "math"
+
+func RoundByDecimals(price float64, decimals int) float64 {
+	precision := math.Pow10(decimals)
+	return math.Round(price*precision) / precision
```

Comment:
what does
	fmt.Println(Round(1.234, -99))
	fmt.Println(Round(1.234, -2))
	fmt.Println(Round(1.234, -1))
	fmt.Println(Round(1.234, 0))
	fmt.Println(Round(1.234, 1))
	fmt.Println(Round(1.234, 2))
	fmt.Println(Round(1.234, 3))
	fmt.Println(Round(1.234, 4))
	fmt.Println(Round(1.234, 99))
return...
tests pls

Reply:
fmt.Println(RoundByDecimals(1.234, -99)) =>    0
fmt.Println(RoundByDecimals(1.234, -2))  =>    0
fmt.Println(RoundByDecimals(1.234, -1))  =>    0
fmt.Println(RoundByDecimals(1.234, 0))   =>    1
fmt.Println(RoundByDecimals(1.234, 1))   =>    1.2
fmt.Println(RoundByDecimals(1.234, 2))   =>    1.23
fmt.Println(RoundByDecimals(1.234, 3))   =>    1.234
fmt.Println(RoundByDecimals(1.234, 4))   =>    1.234
fmt.Println(RoundByDecimals(1.234, 99))  =>    1.2340000000000002

Reply:
fmt.Println(RoundByDecimals(1.234, 99)) => 1.2340000000000002
is not good

I would say:

if the number of requested digits is bigger than the actual digital, use actual
 in this case decimals > 3 {
 decimals = 3
 }